test/js/specs/.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Why two identical `.babelrc` files? Can you not have just one at the top level, in the `js` folder?

Reply:
nope, I don't want auto await in the main.js. only on test code, not on the orchestration which requires finer control